api/models/enums/account_status.js

```diff
@@ -0,0 +1,29 @@
+/**
+ * Enumerated Class for Account Statuses
+ *
+ * None - should never be part of a permission set
+ * It is simply denoting that such a role does not exist.
+ */
+export default class AccountStatus {
+  static Pending = new AccountStatus("pending");
+  static Deactivated = new AccountStatus("deactivated");
+  static Approved = new AccountStatus("approved");
+
+  /**
+ static enum verification method
+ * @param {String} str
+ */
+  static toAccountStatus(str) {
```

Comment:
the verification methods names should correspond to the enum type. Similar for all other files